app/javascript/controllers/image_upload_controller.js

```diff
@@ -34,7 +34,7 @@ export default class extends Controller {
     this.previewTarget.querySelector("img").src = ''
     this.element.classList.remove("show-previews")
     this.contentTarget.focus()
-    window.dispatchEvent(new Event('resize')) // Throw this event will cause textarea_autogrow to reprocess
+    window.dispatchEvent(new Event('resize'))
```

Comment:
I removed the comment because I added another listener for resize so it now does multiple things. It's not worth keeping the comment updated.